python/nano/example/pytorch/inference_pipeline/vision_transformers/vision_transformer.ipynb

```diff
@@ -0,0 +1,952 @@
+    "                   validation_data=val_dataloader,\n",
+    "                   metric=Accuracy(),\n",
+    "                   direction=\"max\",\n",
+    "                   thread_num=4)\n",
+    "acc_model, option = optimizer.get_best_model(accuracy_criterion=0.05)\n",
+    "print(\"best option is \", option)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      " -------------------------------- ---------------------- -------------- ----------------------\n",
+      "|             method             |        status        | latency(ms)  |       accuracy       |\n",
+      " -------------------------------- ---------------------- -------------- ----------------------\n",
+      "|            original            |      successful      |    26.259    |        0.809         |\n",
+      "|           fp32_ipex            |      successful      |    25.588    |    not recomputed    |\n",
+      "|              bf16              |   fail to forward    |     None     |         None         |\n",
+      "|           bf16_ipex            |    early stopped     |   460.863    |         None         |\n",
+      "|              int8              |   fail to convert    |     None     |         None         |\n",
+      "|            jit_fp32            |    early stopped     |   438.965    |         None         |\n",
+      "|         jit_fp32_ipex          |    early stopped     |   445.075    |         None         |\n",
+      "|  jit_fp32_ipex_channels_last   |    early stopped     |   819.841    |         None         |\n",
+      "|         openvino_fp32          |      successful      |    8.677     |    not recomputed    |\n",
+      "|         openvino_int8          |      successful      |    38.01     |        0.772         |\n",
+      "|        onnxruntime_fp32        |      successful      |    19.814    |    not recomputed    |\n",
+      "|    onnxruntime_int8_qlinear    |      successful      |    17.952    |        0.756         |\n",
+      "|    onnxruntime_int8_integer    |   fail to convert    |     None     |         None         |\n",
+      " -------------------------------- ---------------------- -------------- ----------------------\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "optimizer.summary()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Swin Transformer\n",
+    "\n",
+    "[Swin Transformer](https://arxiv.org/abs/2103.14030) proposes hierarchical vision Transformer using shifted windows."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> ⚠️ **Warning**\n",
+    ">\n",
+    "> Swin don't support dynamic batch, so the batch_size of faked_train_dataloader must be the same with val_dataloader.\n",
+    ">\n",
+    "> Otherwise the accuracy will be really slow."
```

Comment:
what do you mean by "accuracy is slow"?

Reply:
sorry, here is a typo. Change to "accuracy will be very low."